backend/tests/core/test_async_loop_issues.py

```
@@ -0,0 +1,320 @@
+    a shared aiohttp ClientSession was bound to one event loop but
+    used from another, causing "Event loop is closed" errors.
+    """
+
+    @pytest.mark.asyncio
+    async def test_session_created_in_current_loop(self) -> None:
+        """Test that session is created in the current event loop.
+
+        This ensures each request gets a session bound to its own loop,
+        avoiding cross-loop issues.
+        """
+        current_loop = asyncio.get_running_loop()
```

Ruff (0.14.14)

[error] Local variable `current_loop` is assigned to but never used

Remove assignment to unused variable `current_loop`

(F841)